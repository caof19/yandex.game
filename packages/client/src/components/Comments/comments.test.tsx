import { render, RenderResult } from "@testing-library/react";
import { Comments, TComment } from ".";

describe.skip("Комментарии", () => {
    let comments: RenderResult;
    const testComments: TComment[] = [
        {
            id: 3,
            topic_id: 14,
            author: "Christensen Lynch",
            createdAt:
                "Wed Sep 09 1970 18:49:41 GMT+0300 (Москва, стандартное время)",
            text: "Incididunt excepteur ex nulla magna adipisicing ipsum sunt eiusmod veniam nisi elit commodo aute. Commodo ea voluptate esse consequat.",
            replies: [],
        },
    ];
    beforeEach(() => {
        comments = render(Comments({ comments: testComments, topic_id: 14 }));
    });
    it("Список тем форума рендерится без ошибок", () => {
        expect(comments).toMatchSnapshot();
    });

    it("Тестовые данные отрисовываются", () => {
        expect(comments.getByText(testComments[0].author)).toBeTruthy();
    });
});
