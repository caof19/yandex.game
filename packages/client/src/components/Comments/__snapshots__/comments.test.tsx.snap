// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Комментарии Список тем форума рендерится без ошибок 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="ant-list ant-list-split css-dev-only-do-not-override-1uq9j6g"
      >
        <div
          class="ant-spin-nested-loading css-dev-only-do-not-override-1uq9j6g"
        >
          <div
            class="ant-spin-container"
          >
            <li
              class="ant-list-item"
            >
              <div
                class="ant-card ant-card-bordered css-dev-only-do-not-override-1uq9j6g"
              >
                <div
                  class="ant-card-head"
                >
                  <div
                    class="ant-card-head-wrapper"
                  >
                    <div
                      class="ant-card-head-title"
                    >
                      Christensen Lynch
                    </div>
                  </div>
                </div>
                <div
                  class="ant-card-body"
                >
                  <div
                    class="ant-flex css-dev-only-do-not-override-1uq9j6g ant-flex-justify-space-between"
                  >
                    <span
                      class="ant-typography css-dev-only-do-not-override-1uq9j6g"
                    >
                      Incididunt excepteur ex nulla magna adipisicing ipsum sunt eiusmod veniam nisi elit commodo aute. Commodo ea voluptate esse consequat.
                    </span>
                    <span
                      class="ant-typography css-dev-only-do-not-override-1uq9j6g"
                    >
                      Wed Sep 09 1970
                    </span>
                  </div>
                </div>
              </div>
            </li>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="ant-list ant-list-split css-dev-only-do-not-override-1uq9j6g"
    >
      <div
        class="ant-spin-nested-loading css-dev-only-do-not-override-1uq9j6g"
      >
        <div
          class="ant-spin-container"
        >
          <li
            class="ant-list-item"
          >
            <div
              class="ant-card ant-card-bordered css-dev-only-do-not-override-1uq9j6g"
            >
              <div
                class="ant-card-head"
              >
                <div
                  class="ant-card-head-wrapper"
                >
                  <div
                    class="ant-card-head-title"
                  >
                    Christensen Lynch
                  </div>
                </div>
              </div>
              <div
                class="ant-card-body"
              >
                <div
                  class="ant-flex css-dev-only-do-not-override-1uq9j6g ant-flex-justify-space-between"
                >
                  <span
                    class="ant-typography css-dev-only-do-not-override-1uq9j6g"
                  >
                    Incididunt excepteur ex nulla magna adipisicing ipsum sunt eiusmod veniam nisi elit commodo aute. Commodo ea voluptate esse consequat.
                  </span>
                  <span
                    class="ant-typography css-dev-only-do-not-override-1uq9j6g"
                  >
                    Wed Sep 09 1970
                  </span>
                </div>
              </div>
            </div>
          </li>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
